charts/opentelemetry-demo/templates/_objects.tpl

```diff
@@ -70,6 +70,12 @@ spec:
           configMap:
             name: {{ include "otel-demo.name" . }}-{{ .name }}-config
       {{- end }}
+      {{- if .initContainers }}
+      initContainers:
+        - name: {{.name}}-init
+          image: busybox:1.28
```

Comment:
Do we need to fix the `busybox` version? Maybe keep it latest to not case security updates etc?

Reply:
Yeah, latest is fine I'd think. Will update.